starfish/core/types/__init__.py

```diff
@@ -15,6 +15,7 @@
 )
 from ._decoded_spots import DecodedSpots
 from ._spot_attributes import SpotAttributes
+from ._spot_finding_resutls import SpotFindingResults
```

Comment:
```suggestion
from ._spot_finding_results import SpotFindingResults
```

(and as such, the file probably needs to be renamed)